packages/web/src/Components/ListItem.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FunctionComponent } from 'react';
+import { withTheme } from '@material-ui/styles';
+import MaterialListItem, {
+  ListItemProps as IListItemProps,
+} from '@material-ui/core/ListItem';
+export { ListItemProps as IListItemProps } from '@material-ui/core/ListItem';
+
+const ListItem: FunctionComponent<IListItemProps> = (props: IListItemProps) => {
+  const { button, ...rest } = props;
+  return button ? (
+    <MaterialListItem button>{props.children}</MaterialListItem>
```

Comment:
Is there a reason for not forwarding the rest of the props when the button is present?

Reply:
When the component has the button prop, when consuming it complains that it cannot have the other props.